vnext/Folly/Folly.vcxproj

```diff
@@ -70,6 +70,16 @@
       <PrecompiledHeader>Create</PrecompiledHeader>
     </ClCompile>
   </ItemGroup>
+  <ItemGroup>
+    <ClCompile Include="$(FollyDir)\folly\ExceptionWrapper.cpp" DisableSpecificWarnings="4541;%(DisableSpecificWarnings)" />
+    <ClCompile Include="$(FollyDir)\folly\Executor.cpp" />
+    <ClCompile Include="$(FollyDir)\folly\Try.cpp" DisableSpecificWarnings="4541;%(DisableSpecificWarnings)" />
+    <ClCompile Include="$(FollyDir)\folly\detail/AsyncTrace.cpp" />
+    <ClCompile Include="$(FollyDir)\folly\executors/InlineExecutor.cpp" />
+    <ClCompile Include="$(FollyDir)\folly\executors/ExecutorWithPriority.cpp" />
+    <ClCompile Include="$(FollyDir)\folly/synchronization\SanitizeThread.cpp" />
```

Comment:
>/ [](start = 41, length = 1)

Please change all forward slashes to the backward slashes. It is better to keep one single style.

Reply:
We no longer have this change..